static/js/apps/nl_eval/search_result.tsx

```diff
@@ -120,18 +120,16 @@ export function SearchResult(props: SearchResultProps): JSX.Element {
     props.overrideStatVars,
   ]);
 
-  // Crop the rankedStatVars to the length of goldenStatVars. Need to evaluate
-  // this further.
-  const evalScore = ndcg(
-    rankedStatVars.slice(0, props.goldenStatVars.length).map((x) => x.dcid),
+  const evalScore = accuracy(
+    rankedStatVars.map((x) => x.dcid),
```

Comment:
Any comments on why this score is better than ncdg?

Reply:
Updated in the PR comment.